src/NCronJob/Registry/JobDefinition.cs

```diff
@@ -120,5 +128,57 @@ public RecurringJobSchedule ToRecurringJobSchedule()
             TimeZone: TimeZone!);
     }
 
+    public void UpdateWith(JobOption? jobOption)
+    {
+        if (jobOption is null)
+        {
+            return;
+        }
+
+        if (jobOption.CronExpression is not null)
+        {
+            UserDefinedCronExpression = jobOption.CronExpression;
+            CronExpression = GetCronExpression(jobOption.CronExpression);
+
+            TimeZone = jobOption.TimeZoneInfo;
+        }
+
+        if (jobOption.Parameter is not null)
+        {
+            Parameter = jobOption.Parameter;
+        }
+
+        if (jobOption.ShouldCrashOnStartupFailure is not null)
+        {
+            ShouldCrashOnStartupFailure = jobOption.ShouldCrashOnStartupFailure;
```

Comment:
Same here

Reply:
The only way to set this is through the `RunAtStartup()` method. And this one always overrides the existing value. There's no way to unset this setting. We should be good I think.

Reply:
Amazing <3